src/components/QuorumPicker.jsx

```diff
@@ -27,151 +29,173 @@ class QuorumPicker extends React.Component {
+    const { requiredSigners, setRequiredSigners } = this.props;
     setRequiredSigners(requiredSigners - 1);
     event.preventDefault();
-  }
+  };
 
   handleIncrementTotalSigners = (event) => {
-    const {totalSigners, setTotalSigners} = this.props;
+    const { totalSigners, setTotalSigners } = this.props;
     setTotalSigners(totalSigners + 1);
     event.preventDefault();
-  }
+  };
 
   handleDecrementTotalSigners = (event) => {
-    const {totalSigners, setTotalSigners} = this.props;
+    const { totalSigners, setTotalSigners } = this.props;
     setTotalSigners(totalSigners - 1);
     event.preventDefault();
-  }
+  };
+
+  render() {
+    const { requiredSigners, totalSigners } = this.props;
+
+    return (
+      <Card>
+        <CardHeader title="Quorum" />
+        <CardContent>
+          <Box>
+            <Grid container justify="center">
+              <Grid container item xs={2} direction="column">
+                &nbsp;
+              </Grid>
+
+              <Grid
+                container
+                item
+                xs={3}
+                direction="column"
+                alignItems="center"
+              >
+                <Grid item>{this.renderIncrementRequiredSigners()}</Grid>
+
+                <Grid item>
+                  <Typography variant="h2">{requiredSigners}</Typography>
+                </Grid>
+
+                <Grid item>
+                  <small>
+                    <p>Required</p>
+                  </small>
+                </Grid>
+
+                <Grid item>{this.renderDecrementRequiredSigners()}</Grid>
+              </Grid>
 
+              <Grid
+                container
+                item
+                xs={2}
+                direction="column"
+                alignItems="center"
+                justify="center"
+              >
+                <Grid item>
+                  <Typography variant="h6">of</Typography>
+                </Grid>
+              </Grid>
+
+              <Grid
+                item
+                container
+                xs={3}
+                direction="column"
+                alignItems="center"
+              >
+                <Grid item>{this.renderIncrementTotalSigners()}</Grid>
+
+                <Grid item>
+                  <Typography variant="h2">{totalSigners}</Typography>
+                </Grid>
+
+                <Grid item>
+                  <small>
+                    <p>Total</p>
+                  </small>
+                </Grid>
+
+                <Grid item>{this.renderDecrementTotalSigners()}</Grid>
+              </Grid>
+              <Grid container item xs={2} direction="column">
+                &nbsp;
+              </Grid>
+            </Grid>
+          </Box>
+        </CardContent>
+      </Card>
+    );
+  }
 }
 
 function mapStateToProps(state) {
   return state.settings;
 }
 
 const mapDispatchToProps = {
-  setTotalSigners,
-  setRequiredSigners,
+  ...settingsActions,
 };
```

Comment:
Here it is again. import * => spread in mapDispatchToProps.

Reply:
Same as above, I'll just search for `import *` and adjust those.